src/core/fair_queue.cc

```diff
@@ -161,7 +161,7 @@ bool fair_queue::class_compare::operator() (const priority_class_ptr& lhs, const
     return lhs->_accumulated > rhs->_accumulated;
 }
 
-fair_queue::fair_queue(fair_group& group, config cfg)
+fair_queue::fair_queue(shared_throttle& group, config cfg)
```

Comment:
Minor: in `fair_queue.hh` the names of parameters of the constructor do not match the ones that are used here. In the header there is `fair_queue(shared_throttle& shared, config cfg)`.

Do we want to keep both places consistent?

Reply:
Yes, I will need to update header name too